Content.Server/Chemistry/ReagentEffects/MakeSentient.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+//using Content.Server.AI.Components; // This was in the original code but caused an error so I'm gonna keep it as a comment just in case
+using Content.Server.Mind.Components;
+using Content.Server.NPC.Components; // Don't think I actually need this but keeping it just in case
+using Content.Shared.Emoting;
+using Content.Shared.Examine;
+using Content.Shared.Movement.Components;
+using Content.Shared.Speech;
+using Content.Shared.Damage;
+
+using Content.Shared.Chemistry.Reagent;
+using Content.Server.Ghost.Roles.Components;
+
+namespace Content.Server.Chemistry.ReagentEffects;
+
+public sealed class MakeSentient : ReagentEffect
+{
+    public override void Effect(ReagentEffectArgs args)
+    {
+        var entityManager = args.EntityManager;
+        var uid = args.SolutionEntity;
+
+        // This makes it so it doesn't affect things that are already sentient
+        if (entityManager.HasComponent<MindComponent>(uid))
+        {
+            return;
+        }
+
+        // This removes the ActiveNPCComponent, because I'm pretty sure that gets rid of the preexisting AI which we probably don't want
+        // Any glitches found in this code probably stem from this part
+        if (entityManager.HasComponent<ActiveNPCComponent>(uid))
+            entityManager.RemoveComponent<ActiveNPCComponent>(uid);
+        
+        // This inactive piece of code makes things able to speak normally, but we probably don't need it
```

Comment:
RemoveComp doesn't require you to check if they have it first. Also, that sounds fun and would make it different than e.g. the random sentience event.

Reply:
Done, got rid of the component checking bits and removed the accent. Pretty sure monkeys have a unique accent meaning this will not let them talk. I can fix this in the future if anyone thinks it's a good idea.